frontend/src/components/list-card/list-card.module.css

```diff
@@ -235,11 +235,13 @@
    * Variants
    */
 
-  .listcard-wrapper.forbidden {
+  .listcard-wrapper.forbidden,
+  .listcard-wrapper.card-not-in-limited-pool {
     filter: grayscale(1);
   }
 
-  .listcard-wrapper.forbidden .name button:not(:hover) {
+  .listcard-wrapper.forbidden .name button:not(:hover),
```

Comment:
I think this would benefit from nesting the `.name button:not(:hover)` selector.